gatk/4.1.7.0R/Dockerfile

```diff
@@ -2,12 +2,31 @@ FROM ubuntu:18.04
 LABEL maintainer="Miguel Brown (brownm28@email.chop.edu)"
 LABEL maintainer="Daniel Miller (millerd15@email.chop.edu)"
 
+ENV PYTHON_VERSION 2.7.15~rc1-1
+ENV WGET_VERSION 1.19.4-1ubuntu2.2
+ENV UNZIP_VERSION 6.0-21ubuntu1
+ENV LIBGOMP1_VERSION 8.4.0-1ubuntu1~18.04
+ENV TABIX_VERSION 1.7-2
+ENV SAMTOOLS_VERSION 1.7-1
+ENV RBASE_VERSION 3.4.4-1ubuntu1
 ENV GATK4_VERSION 4.1.7.0
+ENV OPTPARSE_VERSION 1.6.6
+ENV DATATABLE_VERSION 1.12.8
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt update && apt install -y openjdk-8-jdk python wget unzip libgomp1 tabix samtools r-base-dev; \
+RUN apt update \
+&& apt install -y \
+openjdk-8-jdk \
+python=${PYTHON_VERSION} \
+wget=${WGET_VERSION} \
+unzip=${UNZIP_VERSION} \
```

Comment:
Same as above, might be overkill to set version for this file processing utility